gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/pom.xml

```diff
@@ -58,6 +58,36 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.gravitee.am.factor</groupId>
+            <artifactId>gravitee-am-factor-otp</artifactId>
+            <version>3.10.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.gravitee.am.factor</groupId>
+            <artifactId>gravitee-am-factor-sms</artifactId>
+            <version>3.10.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.gravitee.am.factor</groupId>
+            <artifactId>gravitee-am-factor-email</artifactId>
+            <version>3.10.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.gravitee.am.factor</groupId>
+            <artifactId>gravitee-am-factor-api</artifactId>
+            <version>3.10.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.gravitee.am.gateway.handler</groupId>
+            <artifactId>gravitee-am-gateway-handler-core</artifactId>
+            <version>3.10.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
I think only `gravitee-am-factor-api` and `gravitee-am-gateway-handler-api` with scope `provided` are required here.